capsul/pipeline/pipeline_workflow.py

```diff
@@ -360,10 +360,13 @@ def _replace_dict_transfers(rdict, process, itransfers, otransfers):
         use_input_params_file = False
         if process_cmdline[0] == 'capsul_job':
             python_command = os.path.basename(sys.executable)
+            # pass the sys.path for dev mode
             process_cmdline = [
                 'capsul_job', python_command, '-c',
+                'import sys; sys.path = %s; '
                 'from capsul.api import Process; '
-                'Process.run_from_commandline("%s")' % process_cmdline[1]]
+                'Process.run_from_commandline("%s")' % (sys.path,
```

Comment:
The syntax is confusing here. By luck, the tuple used in the last `%` operator work on the two remaining `%s` in the string. But between these two `%s` there is another `%s` with its `%` operator. This is too difficult to understand. Either, each string with `%s` should have their `%` operator, or the whole string should be within parenthesis and a single `%` operator used with a 3 values tuple.

Reply:
I confess I don't understand your point.
I don't regularly use the C style formatting % operator (I generally prefer the format() method) but I wanted to stay in the coding style existing in this module.
Are you saying that the following syntax is not good? 
`'%s like %s' % ('I', 'strawberries.')`